src/sandbox/MISAGO.md

```diff
@@ -0,0 +1,7 @@
+# Setup and Running Locally
+
+Navigate into the src/sandbox/misago directory and run the following commands:
+
+To start, clone the repository and run ./dev init command in your terminal. This will build necessary docker containers, install python dependencies and initialize the database. After command does its magic, you will be able to start development server using the docker compose up command.
```

Comment:
These instructions need to be more explicit. What command do I run to clone the repository? Which repository am I cloning? I'm sure I could figure it out, but these should be very clear, step by step instructions that don't require Googling or asking for help on Slack. Also, am I going to need need to have Docker? I don't have it installed. How do I do that? We should at least acknowledge the dependency and provided a link to the installation instructions on the Docker website.

Reply:
I've made the instructions more explicit, listing Docker as a prerequisite and providing a link in that case. I've also cleaned up the markup a bit. I've also changed the folder as you suggested to misago-demo.

Reply:
Much clearer, thank you! A few other things that I noticed and went ahead and cleaned up myself are:

- Cleaned up an extraneous /src/sandbox/src/ directory that slipped in somehow.
- Removed the empty /src/sandbox/misago-demo/Misago directory since cloning the repo will create it anyway.
- Added a `cd Misago` step to the instructions and placed the instructions in a code block for easier copying.
- Made a few other minor tweaks to the formatting in MISAGO.md.
- Added /src/sandbox/misago-demo/Misago to .gitignore so that it doesn't end up in the repo by accident later on.

Should be good to go now!